main.go

```diff
@@ -480,6 +496,23 @@ func parseSize(s string) (int64, error) {
 	return n, err
 }
 
+func getDefaultPort() (string, error) {
```

Comment:
Can you add a comment describing what this function does? For example:

> getDefaultPort returns the default serial port depending on the OS.

Reply:
Done.